keps/prod-readiness/sig-apps/1591.yaml

```diff
@@ -1,3 +1,5 @@
 kep-number: 1591
 alpha:
   approver: "@deads2k"
+beta:
+  approver: "@soltysh"
```

Comment:
PRR approver has to be from the "PRR team".

@deads2k - given you reviewed alpha, you may want to take this


BTW - we used to have validation that is checking this - it seems it got broken during refactoring.
@johnbelamaric @ehashman - FYI

Reply:
I'll take a look.

Reply:
It's not being validated because the `latest-milestone` in keps/sig-apps/1591-daemonset-surge/kep.yaml hasn't been updated, so the verify script does not think PRR is needed for beta. Once that updates, this correctly fails the validation.

@ravisantoshgudimetla can you please ensure the KEP metadata is updated for the latest release?

Reply:
Thank you for signing up to review @ehashman

Reply:
I updated the latest-milestone in kep.yaml. PTAL and let me know if you need any other info from me.